### PR TITLE
Add more inputs to GridFieldSetup

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-10233

## Description
The GridFieldSetup is used to populate grid fields and used to have 10 inputs to populate that many columns.  I added 10 more input so 20 columns can be inputed.

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg , please review

File: Packs/CommonScripts/ReleaseNotes/1_15_16.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### GridFieldSetup
+
+Updated the script to included 20 inputs.
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/ReleaseNotes/1_15_16.md:6]:
```suggestion
Updated the script to include 10 additional inputs.
```

**Comment by ShirleyDenkberg** [comment]
@jlevypaloalto Doc review completed.

**Review by jlevypaloalto — APPROVED** [review]
Hey @johnnywilkes, looks great!

Please implement Shirley's suggestions and we should be ready to merge.

**Pull request merged**